File: problem_sixteen.js
// count vowels
var inp ="i am a programer"
function myFunction(inp){
    var value =inp.split(/\s/).join('').split("")
    var count = 0
    for(var i = 0; i < value.length;i++){
        if(value[i ]=="a" || value[i]=="e" || value[i]=="i" || value[i]=="o" || value[i]=="u" ){
            count+=1
        }
       }
    return count   
}
console.log("Number of vowels",myFunction(inp))

File: problem_sixteen.py
# count vowels
inp = input("Enter a sentence:")
def MyFunction(inp):
    val = list(inp.replace(" ", ""))
    count = 0
    for i in val:
        if i == "a" or i == "e" or i =="i" or i =="o" or i =="u":
            count +=1
    return count
print("Number of vowels  =",MyFunction(inp))




